mesh/meshdb.go

```diff
@@ -215,3 +253,52 @@ func (m *meshDB) getLayerMutex(index LayerID) *layerMutex {
 	ll.layerWorkers++
 	return ll
 }
+
+func (m *meshDB) writeTransactions(block *Block) error {
+
+	for _, t := range block.Txs {
+		bytes, err := TransactionAsBytes(t)
+		if err != nil {
+			m.Error("could not write tx %v to database ")
+		}
+
+		//use AccountNonce Recipient Origin as key
```

Comment:
account nonce and recipient is not unique since a transaction can have same nonce and recipient but with different gas cost. I'd recommend using a hash of the entire TX.

Reply:
im using account nonce recipient and origin ...

Reply:
same problem...

Reply:
these three are unique

Reply:
You can, with different gas

Reply:
cool, i changed to sha256 for now